test/java/src/test/java/TestJsonSerializer.java

```diff
@@ -0,0 +1,120 @@
+import java.io.IOException;
```

Comment:
I've got quite a few suggested changes for this file.
Please have a look here:
https://github.com/juliaogris/sysl/blob/json_serializer_mv/test/java/src/io/sysl/reljam/test/tuple/TestJsonSerializer.java

This also contains the `BigDecimal` fix, object comparison and simplified Exception handling